node/storagenode.go

```diff
@@ -251,24 +251,22 @@ func (node *Node) readFromStorage() {
 	// Open the file for reading
 	file, err := os.OpenFile(filePath, os.O_RDONLY|os.O_CREATE, 0666)
 	if err != nil {
-		fmt.Printf("Error opening or creating the file for reading: %v\n", err)
+		log.Error().Err(err).Msg("Error opening or creating the file for reading")
 		return
 	}
 	defer file.Close()
 	var storage map[uint64]map[uint64][]string
 	decoder := json.NewDecoder(file)
 	err = decoder.Decode(&storage)
 	if err != nil {
-		fmt.Printf("Error decoding the JSON data: %v\n", err)
+		log.Error().Err(err).Msg("Error decoding the JSON data")
 		return
 	}
 
-	if node.Logging {
-		fmt.Printf("Data read from file\n")
-	}
+	log.Info().Msgf("JSON data read from file: %s", filePath)
 
 	for key, value := range storage {
-		fmt.Printf("Key: %v, Value: %v\n", key, value)
+		log.Info().Msgf("Key: %v, Value: %v\n", key, value)
```

Comment:
I think this can also be moved to DEBUG... the output from here is huge and moves the terminal way too fast.